src/mobile/src/ui/routes/entry.js

```diff
@@ -203,7 +203,7 @@ onAppStart()
             const { settings: { versions, completedMigration } } = reduxState;
 
             if (
-                versions.buildNumber < 41 &&
+                versions.buildNumber < 42 &&
```

Comment:
This should stay at 41, otherwise it will trigger another migration in a subsequent update

Reply:
Na it should be `< 42` because that means Realm migration will trigger for this version

Reply:
Ooh my bad, I didn't realize that these held the current version and not the previous one